smartmirror.py

```diff
@@ -10,8 +10,7 @@
 import feedparser
 from PIL import Image, ImageTk
 
-ip = '<IP>'
-country_code = 'us'
+country_code = None  # set this here if you don't want news from your current location
 weather_api_token = '<TOKEN>'
```

Comment:
Sorry, I just realized that I phrased my previous comment wrong. country_code was always overridden in weather. I would have preferred that that was some logic that used the default value if the user chooses to set it (for example, if they changed None to `us` it would always use `us`). These features were added in #4 and #6. 

If you want to add anything else, I can merge them in.